packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -20,6 +20,16 @@
   background-color: var(--color-white);
 }
 
+.va-button-primary--alternate {
+  background: var(--color-green);
+
+  &:hover,
+  &:focus {
+    background-color: var(--color-green-dark);
```

Comment:
If this is supposed to be replicating the [Formation `.va-button-primary` style](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_va.scss#L1046-L1062), shouldn't this be `color-green-darker` (#195c27)?

https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-variables.scss#L105

Reply:
I added darker in the latest push